### PR TITLE
Create Dockerfile and basic instructions

### PR DESCRIPTION
**Pull request by singiamtel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +29,17 @@ It is based on [Discord.Net](https://github.com/discord-net/Discord.Net) and [SK
 10) Download the **core** you want to use [here](https://buildbot.libretro.com/nightly/) and place it in the **cores folder**
 11) Place your **game rom** in the netcoreapp3.1 folder and run the bot
 
+Alternatively you can use Docker with the provided Dockerfile
+
+Steps 1 to 6 are the same as above
```
Comment (screenshakes) [inline README.md:34]: All those lines don't seem to break when the markdown is rendered

File: README.md [hunk 2]
```diff
@@ -27,6 +29,17 @@ It is based on [Discord.Net](https://github.com/discord-net/Discord.Net) and [SK
 10) Download the **core** you want to use [here](https://buildbot.libretro.com/nightly/) and place it in the **cores folder**
 11) Place your **game rom** in the netcoreapp3.1 folder and run the bot
 
+Alternatively you can use Docker with the provided Dockerfile
+
+Steps 1 to 6 are the same as above
+7) Create a folder called `ROMs/` in the project root and place your game ROM in it
+8) Create a folder called `cores/` in the project root and place the libretro core ([download it here](https://buildbot.libretro.com/nightly/)) in it too
+9) Create a text file called `token` in the project root and paste your bot token in it
+10) ``docker build -e ROM='your_rom_name' -e CORE='your_core_name' -t dew .``. The core name should be specified without the ``_libretro.so`` extension, and the ROM name should be specified without the extension
```
Comment (screenshakes) [inline README.md]: Could you change it to something like "run 'docker ...' in your terminal"?
It's obvious but better be clear for everyone

File: README.md [hunk 3]
```diff
@@ -27,6 +29,17 @@ It is based on [Discord.Net](https://github.com/discord-net/Discord.Net) and [SK
...
 11) Place your **game rom** in the netcoreapp3.1 folder and run the bot
 
+Alternatively you can use Docker with the provided Dockerfile
+
+Steps 1 to 6 are the same as above
+7) Create a folder called `ROMs/` in the project root and place your game ROM in it
+8) Create a folder called `cores/` in the project root and place the libretro core ([download it here](https://buildbot.libretro.com/nightly/)) in it too
+9) Create a text file called `token` in the project root and paste your bot token in it
+10) ``docker build -e ROM='your_rom_name' -e CORE='your_core_name' -t dew .``. The core name should be specified without the ``_libretro.so`` extension, and the ROM name should be specified without the extension
+11) ``docker run dew`` should now start the bot. If you want to run the bot in the background, use ``docker run -d dew``
```
Comment (screenshakes) [inline README.md]: Same

**Comment by screenshakes** [comment]
Hey, thanks a lot for this PR!
I've only added a few comments to the readme changes

File: README.md (hunk 1 above)
Comment (singiamtel) [reply README.md:34]: Oh oops

**Pull request merged**